sql/core/src/test/resources/sql-tests/inputs/create.sql

```diff
@@ -0,0 +1,23 @@
+-- Catch case-sensitive name duplication
+SET spark.sql.caseSensitive=true;
+
+CREATE TABLE t(c0 STRING, c1 INT, c1 DOUBLE, c0 INT) USING parquet;
```

Comment:
In `DDLSuite`, we already have simple tests for duplicate columns. we better moving these tests there?